src/atoms/cc-input-text.js

```diff
@@ -50,7 +50,7 @@ function arrayEquals (a, b) {
  * @event {CustomEvent} cc-input-text:requestimplicitsubmit - Fires when enter key is pressed in simple mode, in tags mode or when ctrl+enter is pressed in multi mode.
  * @event {CustomEvent<string[]>} cc-input-text:tags - Fires an array of tags whenever the `value` changes (separated by spaces).
  *
- * @cssprop {FontFamily} --cc-input-font-family - The font-family for the input content (defaults: `inherit`).
+ * @cssprop {FontFamily} --cc-input-font-family - The font-family for the input content (defaults: `--cc-ff-monospace` from the them).
```

Comment:
@hsablonniere ahh it's a bit more complicated than this actually.
It only defaults to `--cc-ff-monospace` if you use tags. If not, it inherits.
I'll update the desc if that's ok with you ?

Reply:
Nice :+1: